scilpy/tracking/tools.py

```diff
@@ -36,23 +33,23 @@ def filter_streamlines_by_length(streamlines,
         List of filtered streamlines by length.
 
     filtered_data_per_point: dict
-        dict of data per point for filtered streamlines
+        dict of data per point for filtered streamlines.
     filtered_data_per_streamline: dict
-        dict of data per streamline for filtered streamlines
+        dict of data per streamline for filtered streamlines.
     """
 
     lengths = []
-    for streamline in streamlines:
+    for streamline in sft.streamlines:
```

Comment:
Could be replaced by a np.apply_along_axis (gives you a numpy array for free as output also)